lib/util/git/set-branch-commit.ts

```diff
@@ -0,0 +1,30 @@
+import { getCache } from '../cache/repository';
+import type { BranchCache } from '../cache/repository/types';
+import { getBranchCommit } from '.';
+
+export function setBranchCommit(
+  branchName: string,
+  baseBranch: string,
+  commitSha: string
+): BranchCache {
+  const cache = getCache();
+  cache.branches ??= [];
+  let branch = cache.branches.find((br) => br.branchName === branchName);
+  if (!branch) {
+    branch = {
+      branchName: branchName,
+      baseBranch: baseBranch,
```

Comment:
```suggestion
      branchName,
      baseBranch,
```

I thought our automated linting would have picked this up

Reply:
Local system is slow so sometimes linter misbehaves, github checks should have picked up. I am not sure either.🤷‍♂️

Reply:
Btw what is this behaviour called? Want to look for the setting in linter that detects it.

Reply:
I'm not sure actually. Essentially it's a redundant check. Maybe our linter ignores it now? I'd like to add it back if so.

Reply:
It's called _shorthand property names_. [MDN link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#syntax)

Reply:
https://github.com/renovatebot/renovate/pull/18080